docs/developers_guide.rst

```diff
@@ -85,6 +85,16 @@ This is required if there were any JS changes:
 - from local directory (not in lms shell)
 - ``npm run build``
 
+Hot reload JS
```

Comment:
I had a fair bit of trouble finding where verison of the files were in the browser developer tools. Bonus points for adding docs on how to do that. Also notes around port collisions, since my dev server defaulted to `8080` which docker already uses.